superset/assets/javascripts/explore/stores/visTypes.js

```diff
@@ -517,6 +517,44 @@ export const visTypes = {
     ],
   },
 
+  deck_polygon: {
+    label: t('Deck.gl - Polygon'),
+    requiresTime: true,
+    controlPanelSections: [
+      {
+        label: t('Query'),
+        expanded: true,
+        controlSetRows: [
+          ['polygon', 'row_limit'],
```

Comment:
We may want to support the `reverse` control here. Seems like so far I've seen both lat/long long/lat in different places.